Documents/CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+* Limit the first line to 72 characters or less.
+* Reference issues and pull requests liberally after the first line.
+
+### Naming Files and Folders:
+
+When (re)naming files and folders always do so using ['PascalCase'](https://techterms.com/definition/pascalcase), this makes naming consistant which helps with organization and clarity.
+
+E.X. "LockerDoorAnimationController.file" would be correct,
+not "Locker Door Animation Controller.file",
+not "lockerdoor animation controller.file",
+not "AnimationController - LockerDoor.file".
+
+### Game Testing & Reporting Bugs:
+
+Before reporting a bug, please check the reported [bugs on our issues page](https://github.com/RE-SS3D/SS3D/labels/Bug) to see if the bug has already reported.
+
+Explain the issue clearly and include any additional details to help maintainers reproduce the problem:
+
+* **Use a clear and descriptive title** for the issue to identify the problem.
+* **Describe the steps to reproduce the problem** as specifically and detailed as possible.
+* **Describe the behavior you observed** and point out the problem with that behavior.
+* **Describe the expected behavior** that should happen instead.
+* **Include pictures/videos** to show the steps taken to demonstrate the problem.
+* **Include your game version and OS** to help track if the problem is version specific.
+* **Include your game settings** to help track if the problem is settings specific.
+* **If you're reporting a game crash**, include a crash report with the error log. It's location depends on your operating system, so follow [this official guide](https://docs.unity3d.com/Manual/LogFiles.html).
+* **If the problem isn't triggered by a specific action**, describe what you were doing before the problem happened.
+
+## Project Maintainers
+
+The set of people responsible for making sure the contribution cycle runs smoothly and that the changes made to the project are wanted, correct, and functional. On the Github these people are labeled "members", and on the Discord they are known as "Centcom".
+
+The project maintainers have the following goals:
+
+* To prioritize issues and pull requests so the most important changes get focused on and added to the project first.
+* To make sure no contributions, however minor, don't get overlooked. Despite some contributions having greater priority, all contributions are created equal.
+* To verify that contributions comply with our standards, will not break the project, and are wanted, before they are accepted into the project.
+
+## Setting Up & Running The Project
+
+### Downloading Unity:
+
+Option A.
+1. Clone this (SS3D) repository.
+2. Download [Unity Hub](https://unity3d.com/get-unity/download).
+3. Download the current Unity version that the project tells you.
+
+Option B.
+1. Clone this (SS3D) repository.
+2. [Download Unity 2019.4.30f1 or newer](https://unity3d.com/get-unity/download/archive) and open the folder containing the project in Unity.
```

Comment:
Incorrect version - 2021.3.0f1 now required I believe.

Reply:
Thanks for the review! Not sure how I missed those.